### ISSUE TITLE
Alwas getting System UI isnt responding error on Android emulator

### ISSUE THREAD

**Issue by vytautas-pranskunas-** [flutter/flutter#90941]
I have this issue with my current project that after certain amount of time using application on Android simulator, then leaving it idle (do not sot know exact recepe to get this behaviour) I am starting to receive "System UI isn't responding" message. If i choose "Close app" app just repoens and works again. But this message keeps poping from time to time until I do cold reboot of emulator. And it starts againa after some time. I have raised ticket before but there was no outcome because i could not provide demo app so i left it like this restarting simulators.
Recently I have created new simple application with 3 Pages just for demo purposes and running it on Tablet emulator on Android and it started again.... 
Now I am sure that that was not a problem of application but it starts on every application because my demo app is doing nothing.. it just have 3 tabs  with 3 different pages and fake login page which redirects to main one.

![Screenshot 2021-09-29 at 11 42 10](https://user-images.githubusercontent.com/3918999/135245331-2258e627-7671-456f-9176-86aa500bd3f9.png)

Flutter doctor:
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.5.1, on macOS 11.5.2 20G95 darwin-x64, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.2)
[✓] VS Code (version 1.60.2)
[✓] Connected device (4 available)
```

**Comment by danagbemava-nc** [comment]
Hi @vytautas-pranskunas, 

Please provide a [minimal complete reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example).

Also, what specs does the emulator have? 

Do you also experience this with an actual android device? 

Thank you

**Comment by vytautas-pranskunas-** [comment]
hi, I am not experiencing it with any real device (at least in prod build - not sure about debug mode). Just with any android simulator
[Archive.zip](https://github.com/flutter/flutter/files/7252432/Archive.zip)
.

**Comment by danagbemava-nc** [comment]
Can you try this, just open the emulator without running your flutter app. Try using it like a normal device and see if you run into this issue. 

Sometimes, I ran into similar issues with my emulator, in that case, I usually delete the emulator and create a new one and it stops.

**Comment by vytautas-pranskunas-** [comment]
I do not think that creating new emulator will work because few days ago i have added new tablet simulator from scratch and it started there after running app.

**Comment by danagbemava-nc** [comment]
What were the specs of the emulator you created?

**Comment by vytautas-pranskunas-** [comment]
These are all simulators that I use. Last one Nexus has been created recently. 
However i see this issue in all of them. I do not think that it is emulator specific. There should be something in the core.

![Screenshot 2021-09-30 at 13 51 01](https://user-images.githubusercontent.com/3918999/135450137-e3ab3d97-3979-4f75-aa30-b1dd1f95a806.png)

**Comment by danagbemava-nc** [comment]
The reason I don't believe it is a flutter issue is because this issue has existed long before flutter and it isn't an issue that only flutter developers face. 

The common denominator here aside flutter happens to the the `API level` you're using, you can also try creating a new emulator with a different `API level` to see if you still experience it. 

Kindly check these SO threads to see if any of them solves your issue. 

https://stackoverflow.com/questions/63371056/the-system-ui-isnt-responding-in-android-emulator-flutter
https://stackoverflow.com/questions/65295204/android-studio-4-1-1-system-ui-isnt-responding-error
https://stackoverflow.com/questions/43779596/process-system-isnt-responding-in-android-emulator
